SU2_CFD/src/solvers/CTurbSSTSolver.cpp

```diff
@@ -410,6 +410,17 @@ void CTurbSSTSolver::BC_HeatFlux_Wall(CGeometry *geometry, CSolver **solver_cont
 
     const auto iPoint = geometry->vertex[val_marker][iVertex]->GetNode();
 
+
+    /*--- Evaluate nu tilde at the closest point to the surface using the wall functions. ---*/
+
+    if (config->GetWall_Functions()) {
+      SU2_OMP_MASTER
+      SetNuTilde_WF(geometry, solver_container, conv_numerics, visc_numerics, config, val_marker);
+      SU2_OMP_BARRIER
+      return;
+    }
```

Comment:
@pcarruscag I have to confess that I do not know how openmp works, I'm not using it. This piece of code fails. This code was already there, so this was never working properly I guess? Do you know why this fails?

Reply:
You need to move it out of the iVertex loop.

Reply:
ok, that's what I suspected. Will do that...